### ISSUE TITLE
[Select] Option group label obscured

### ISSUE THREAD

**Issue by dmcalester** [RocketCommunicationsInc/astro#406]
## Expected Behavior
Option group labels text should be legible achieving a minimum of WCAG 2.1 AA contrast compliance

## Actual Behavior
Option group labels are hard or impossible to read.

https://astro-components.netlify.app/?path%3D/story/forms-select-menu--with-option-groups

Affected platforms appear to be

Mac Firefox
<img width="332" alt="Screen Shot 2022-05-13 at 10 51 02 AM" src="https://user-images.githubusercontent.com/60529018/168341361-7ada4972-359a-4b89-9f00-da1b85caeba3.png">

Windows Edge (Chromium)
<img width="332" alt="Screen Shot 2022-05-13 at 10 51 02 AM" src="https://user-images.githubusercontent.com/60529018/168341468-6a6e483b-a80d-403c-83dc-0dac08d1d2b3.png">

Window Chrome
<img width="290" alt="Screen Shot 2022-05-13 at 11 05 05 AM" src="https://user-images.githubusercontent.com/60529018/168341882-5c7fcf38-747a-4d78-b5d9-7e22f3e9e2f6.png">

Windows Firefox
<img width="330" alt="Screen Shot 2022-05-13 at 11 07 33 AM" src="https://user-images.githubusercontent.com/60529018/168342205-68870a6d-4628-4cfb-bc34-5ad4d0494833.png">
